lang/image.bzl

```diff
@@ -148,7 +148,7 @@ def _app_layer_impl(ctx, runfiles = None, emptyfiles = None):
     parent_parts = _get_layers(ctx, ctx.attr.name, ctx.attr.base)
     filepath = _final_file_path if ctx.attr.binary else layer_file_path
     emptyfilepath = _final_emptyfile_path if ctx.attr.binary else _layer_emptyfile_path
-    dep = ctx.attr.dep or ctx.attr.binary
+    dep = (ctx.attr.dep or ctx.attr.binary)[0]
```

Comment:
It seems like indexing at `0` will also be applied to `ctx.attr.dep`. Why did this change need to be accounted for `dep` object as well as the `binary` object?

Reply:
Because both dep and binary now use a transition configuration.